microprot/scripts/calculate_Neff.py

```diff
@@ -106,8 +111,11 @@ def _calculate_Neff(infile, outfile, cutoff):
     """
     msa = parse_msa_file(infile)
     hdm = hamming_distance_matrix(msa)
-    clu = cluster_sequences(hdm, cutoff)
-    Neff = effective_family_size(clu, msa.shape[1])
+    try:
+        clu = cluster_sequences(hdm, cutoff)
+        Neff = effective_family_size(clu, msa.shape[1])
+    except ValueError:
+        Neff = 0.0
```

Comment:
wouldn't it be better to put this clause in appropriate function, instead of the click interface?
If someone uses function directly, this does not solve the problem.